guide/private-key-management/backups.md

```diff
@@ -0,0 +1,256 @@
+   mobileRetina = "/assets/images/guide/private-key-management/backups/backups-mobile@2x.jpg"
+   alt-text = "Several devices exchanging data"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Bitcoin backups
+
+The self-custody of bitcoin is an important right, but also a financial responsibility that can scale from pocket money to life savings, and take practice getting used to.
+
+This guide is a straightforward starting point for simple bitcoin backups, aimed at users of bitcoin products.
+
+If you are a designer or developer, make sure you offer guidance to your users so they can secure their bitcoin with an appropriate backup.
+
+
+## What are backups?
+
+When we talk about backups, we mean saving the information needed to restore access to a wallet outside of the application that created the private keys. Often, this will just be the [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) of 12 or 24 words, but can also include other information like a *pass-phrase* and [derivation path]({{ '/guide/glossary/#derivation-path' | relative_url }}).
+
+The main risks that backups need to protect against are:
+
+- Loss of wallet - you lost the device and application holding the private key
+- Remote theft - your backup got discovered online by a bad actor
+- Physical damage - your backup got destroyed
+- Theft on discovery - your backup got discovered in person by a bad actor
+- Catastrophic events - your backup location got destroyed
+- Inheritance loss - next-of-kin were unable to access your wallet
+
+
+## Backup levels, basic to advanced
+
+A basic backup can be done in a few minutes, while more advanced setups can require more time and effort, like the purchase of dedicated material. 
+
+Let’s look at some real life examples of backup methods for single-key wallets and what they protect against.
+
+<br>
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/backups/backup-table.png"
+   retina = "/assets/images/guide/private-key-management/backups/backup-table@2x.png"
+   alt-text = "Table of backup levels"
+   width = 904
+   height = 625
+   layout = "full-width"
+%}
+
+{% include tip/recommendation.html %}
+
+Level 0 and 1 are good when getting started, and require very little effort.
+
+Level 2, 3 or 4 are recommended for storing larger amounts.
+
+{% include tip/close.html %}
+
+<br>
+
+#### Level 0
+## Encrypted cloud backup 
+
+By storing the recovery-phrase in an online location that is encrypted (not in plain text) and hard to access by anyone other than yourself, you are protected against *loss of wallet*. This is the absolute minimum you should do and only takes a few minutes.
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/backups/cloudbackup.png"
+   retina = "/assets/images/guide/private-key-management/backups/cloudbackup@2x.png"
+   alt-text = "Encrypted cloud backup"
```

Comment:
Make sure to add width and height attributes to all images on the page.

Reply:
Fixed